designs/virtualization/4.6/snapshots/snapshots.md

```diff
@@ -0,0 +1,62 @@
+---
+parent: Virtualization
+version: 4.6
+---
+
+# VM snapshots
+
+This item will cover updates to vm-snapshots
+- Creating a VM snapshot
+- Snapshots on the VM details page
+- Restoring a VM from snapshot
+
+## Creating a snapshot
+
+![Virtualization - VM snapshots tab](img/0-0.jpg)
+From the virtual machine details  page > snapshots tab, the user selects Take Snapshot”.
+
+![Virtualization - take snapshot](img/0-1.jpg)
```

Comment:
I talked to @aglitke and @glekner about this and I think we'd like to have a list of the of disks included here (and also the ones that won't be included). 
I filed a bug for this too 
https://bugzilla.redhat.com/show_bug.cgi?id=1865870

Reply:
according to @jelkosz this is something that we cannot do, at least for this 4.6 quick and lite version. I will try to push it on the 4.x (the MD was removed for now. I will launch it again at a different PR as the doc is stabilized)